### ISSUE TITLE
Setting form params broken when .back().body() called in the meantime

### ISSUE THREAD

**Issue by golszewski86** [jcabi/jcabi-http#243]
The following code:

```java
new JdkRequest("http://localhost/")
    .body()
    .formParam("foo1", "bar1")
    .back()
    .body()
    .formParam("foo2", "bar2")
    .back()
    .body()
    .formParam("foo3", "bar3")
    .formParam("foo4", "bar4")
    .back()
```

Creates the following request parameters:

`foo1=bar1foo2=bar2foo3=bar3&foo4=bar4`

As you can see, the ampersand is missing after `bar1` and after `bar2`.

Tested on jcabi-jdbc 1.17.5

**Comment by andreoss** [comment]
Fixed in [1.17.6](https://search.maven.org/artifact/com.jcabi/jcabi-http/1.17.6/jar)

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-http#243` is not assigned, can't get performer
